tests_for_Notify.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from Notification import Notification
+
+class Notification_Test(unittest.TestCase):
+  def test_notify_to_controller(self):
+    NotifyTest1 = Notification("TO_CONTROLLER","TOO_LOW")
+    NotifyTest2 = Notification("TO_CONTROLLER","TOO_HIGH")
+    NotifyTest3 = Notification("TO_CONTROLLER","NORMAL")
+    self.assertTrue(NotifyTest1.notify())

```

Comment:
Try changing the tests to reflect the user-need. The user's need isn't that notification must return true... it's that the notification content is correct. Imagine yourself as the recipient of the email. What would you want the subject to say? What should the 'from' address be? Write tests for that and satisfy them in the code.